mailbox/caching/src/main/java/org/apache/james/mailbox/caching/CachingMailboxMapper.java

```diff
@@ -1,9 +1,11 @@
 package org.apache.james.mailbox.caching;
```

Comment:
It's not your fault but a license is missing here too...
Could you add it in a separate commit, please ?